source/connect/authentication-connection.txt

```diff
@@ -13,3 +13,162 @@ Authentication Connection Tab
+
+      - (Optional) :manual:`Authentication Database </core/security-users/#authentication-database>`
+
+      - Authentication Mechancism:
+         
+        - Default
+
+          The ``Default`` setting selects the first authentication mechanism 
+          supported by the server according to an order of preference. For 
+          more information on the order of preference, see :driver:`Default 
+          </node/current/fundamentals/authentication/mechanisms/#default>`
+
+        - :manual:`SCRAM-SHA-1 </core/security-scram/>`
+
+        - :manual:`SCRAM-SHA-256 </core/security-scram/>`
+
+      .. _x509:
+
+      X.509
+      ~~~~~
+
+      Select :guilabel:`X.509` if the deployment uses :manual:`X.509 
+      </core/security-x.509/>` as its authentication mechanism.
+
+      .. note:: 
+
+         X.509 Authentication requires a client certificate. To enable 
+         TLS and add a client certificate, see the :ref:`TLS / SSL tab <tls-ssl-tab>`
+         in |compass-short|.
+
+      .. _kerberos-authentication:
+
+      Kerberos
+      ~~~~~~~~
+
+      Select :guilabel:`Kerberos` if the deployment uses :manual:`Kerberos 
+      </core/kerberos/>` as its authentication mechanism. 
+
+      Provide the following information:
+
+      - Principal
+
+      - (Optional) Service Name
+
+      - (Optional) Canonicalize Host Name
+
+        |
+
+        When you enable this setting, Kerberos uses the canonicalized form of 
+        the host name ``(cname)`` when constructing the principal for |compass|.
+
+        |
+
+        You have the following options: *waiting on further tech info*
+
+        .. list-table::
+           :header-rows: 1
+           :widths: 50, 50
+
+           * - Option
+             - Description 
+
+           * - Forward
+             - DNS lookup that uses domain name to find IP address.
+
+           * - Forward and Reverse
+             - *waiting on further tech clarification*
+
+        |
+
+        For more information on principal name canonicalization in Kerberos, see 
+        this `RFC document <https://tools.ietf.org/html/rfc6806.html>`__.
+
+      - (Optional) Service Realm 
+  
+        The service realm is the domain over which a Kerberos authentication 
+        server has the authority to authenticate a user.
+  
+      - (Optional) Provide password directly
+      
+      .. _ldap-authentication:
+
+      LDAP
+      ~~~~
+
+      Select :guilabel:`LDAP` if the deployment uses :manual:`LDAP 
+      </core/security-ldap-external/>` as its authentication mechanism. 
+
+      Provide the following information:
+
+      - Username
+
+      - Password
+
+      .. _aws-iam-authentication:
+
+      AWS IAM  
+      ~~~~~~~
+
+      Select :guilabel:`AWS IAM` if the deployment uses `AWS IAM 
+      <https://docs.aws.amazon.com/IAM/latest/UserGuide/introduction.html>`__ as 
+      its authentication mechanism. 
+      
+      Provide the following information:
+
+      - (Optional) AWS Access Key Id
```

Comment:
Just noting here that I'd like further clarification on why the access / secret keys are optional. Not a blocker for copy review.

Reply:
Done.